<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>K3bManualBitrateSettingsWidget</class>
 <widget class="QWidget" name="K3bManualBitrateSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>309</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Quality</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="m_radioConstantBitrate">
        <property name="text">
         <string>Constant Bitrate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KComboBox" name="m_comboConstantBitrate"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="m_radioVariableBitrate">
        <property name="text">
         <string>Variable Bitrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_frameVariableBitrate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QCheckBox" name="m_checkBitrateMaximum">
           <property name="text">
            <string>Maximum bitrate:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="KComboBox" name="m_comboMaximumBitrate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="m_checkBitrateAverage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Average bitrate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="m_spinAverageBitrate">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="suffix">
            <string> kbps</string>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>310</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="m_checkBitrateMinimum">
           <property name="text">
            <string>Minimum bitrate:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="KComboBox" name="m_comboMinimumBitrate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Channel Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="KComboBox" name="m_comboMode">
        <property name="toolTip">
         <string>Select the channel mode.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;Select the channel mode of the resulting Mp3 file:
&lt;p&gt;&lt;b&gt;Stereo&lt;/b&gt;&lt;br&gt;
In this mode, the encoder makes no use of potential correlations between the two input channels; it can, however, negotiate the bit demand between both channel, i.e. give one channel more bits if the other contains silence or needs fewer bits because of a lower complexity.
&lt;p&gt;&lt;b&gt;Joint-Stereo&lt;/b&gt;&lt;br&gt;
In this mode, the encoder will make use of correlations between both channels. The signal will be matrixed into a sum (&quot;mid&quot;), computed by L+R, and difference (&quot;side&quot;) signal, computed by L-R, and more bits are allocated to the mid channel. This will effectively increase the bandwidth if the signal does not have too much stereo separation, thus giving a significant gain in encoding quality.
&lt;p&gt;&lt;b&gt;Mono&lt;/b&gt;&lt;br&gt;
The input will be encoded as a mono signal. If it was a stereo signal, it will be downsampled to mono. The downmix is calculated as the sum of the left and right channel, attenuated by 6 dB.</string>
        </property>
        <item>
         <property name="text">
          <string>Stereo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joint Stereo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mono</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_radioConstantBitrate</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_comboConstantBitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBitrateMaximum</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_comboMaximumBitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBitrateAverage</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_spinAverageBitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBitrateMinimum</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_comboMinimumBitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioVariableBitrate</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_frameVariableBitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
